Casks/forscan.rb

```diff
@@ -59,7 +59,7 @@
     end try
 
     -- Finally open the FORScan application itself through Wine
-    do shell script "open -a 'Wine Stable' \\"$HOME/.wine/drive_c/Program Files (x86)/FORScan/FORScan.exe\\""
+    do shell script "/opt/homebrew/bin/wine $HOME'/.wine/drive_c/Program Files (x86)/FORScan/FORScan.exe'"
```

Comment:
Let's try this instead:
```suggestion
    do shell script "open -a 'Wine Stable' '~/.wine/drive_c/Program Files (x86)/FORScan/FORScan.exe'"
```